### DISCUSSION TITLE
New empirical evidence that falsify classical gravity (via GAIA)

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#263] [Astrophysics and Physics]
https://iopscience.iop.org/article/10.3847/1538-4357/ad61e9
https://phys.org/news/2024-09-gravitational-anomaly-favor-gravity.html
Note that Chae previously published similar results but via a different method.
There are also analysises from different researchers that rule out most (but not necessarilly all) MOND theories, in either case, classical gravity is clearly empirically falsified.
Also studies that corroborate MOND often can be explained by alternative non MONDian theories, such as MOG gravity or modified/quantized inertia and maybe, Lesage's theory or the sagnac effect or massive gravity or aether tensor scalar theory

**Comment by HanDeBruijn** [comment]
Related: ["Breakdown of the Newton–Einstein Standard Gravity at Low Acceleration in Internal Dynamics of Wide Binary Stars"](https://github.com/orgs/a-cosmology-group/discussions/122). My [first attempt](https://github.com/orgs/a-cosmology-group/discussions/122#discussioncomment-6708514) to tackle the problem is in that thread.

### Prerequisites

- First read a contribution in [thread 262](https://github.com/orgs/a-cosmology-group/discussions/262#discussioncomment-10606287). The [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) is adopted as usual.

![Klokjes](https://github.com/user-attachments/assets/34e56fbf-3db8-4091-812a-93e292826653)
There is more than **atomic clock**s in the universe. In our common lives, time is rather measured in terms of a day (rotation of the earth) and a year (the time it taks for the earth to complete its orbit around the sun). According to orbital mechanics (e.g. a circular orbit) we find for an (infinitesimal) **orbital clock** period $dT$ the following.

$$
m \omega^2 r = G\frac{m M}{r^2} \quad \Longrightarrow \quad \omega = \sqrt{G\frac{M}{r^3}} \quad \Longrightarrow \quad dT \sim 2\pi\sqrt{\frac{r^3}{GM}}
$$

where $\omega=$ angular frequency, $r=$ radius of orbit, $m,M=$ mass, $G=$ gravitational constant.
A crucial assumption is that **space is static and Euclidean**; there is NO general relativity to distort it. And so orbital **time** is the only quantity that can be affected by our Variable Mass. Futhermore we see that a second in orbital time $T$ is not inversely proportional to mass, but inversely proportional to the **square root** of it.

$$
\frac{dT}{dT_0} = \sqrt{\frac{m_0}{m}} \quad \mbox{contrary to} \quad \frac{dt}{dt_0} = \frac{m_0}{m}
$$

- Then read a follow up in [thread 122](https://github.com/orgs/a-cosmology-group/discussions/122#discussioncomment-10398651).

### Calculation

Now we are ready to tackle the wide binaries problem from a somewhat different angle, namely with help of this formula (in thread 122):

$$
\vec{a}_T = \vec{a}_t - \vec{v}_t/A \quad \mbox{or} \quad \vec{a}_t = \vec{a}_T + \vec{v}_T/A
$$

augmented with $A=2/H$ , giving as a first approximation for a circular orbit with separation $s$ :

$$
a_t = a_T + \frac{1}{2}H.v_T \quad \Longrightarrow \quad a = \frac{v^2}{s} + \frac{1}{2}H.v = \frac{v^2}{s}\left(1+\frac{H}{2v/s}\right)
$$

In the [article](https://iopscience.iop.org/article/10.3847/1538-4357/ad61e9) we read about a boost factor 1.37 :

$$
1+\frac{H}{2v/s} = 1.37 \quad \Longrightarrow \quad v = H/2\times s/0.37
$$

and wide binaries not obeying Newton seem to have a separation greater than s &gt; 5 kAU.
```
# 1 megaParsec
Mpc := 3.08567758*10^22;
# Hubble parameter
H := 73.4*1000/Mpc;
# Astronomical Unit
AU := 149597870700;
# Separation of binary
s := 5*1000*AU;
# Boost factor 1.37 ==>
v := H/2*s/0.37;
#                v := .2404413798e-2 = 2.4 mm/s
# Acceleration
a := v^2/s*1.37;
```
Resulting in an (observed) acceleration of $a = 0.1058872267\times 10^{-19}\mbox{ }m/s^2$ .
Which is orders of magnitude different from a nanometer per second squared. Someone has made a mistake somewhere ..

**Reply by HanDeBruijn** [reply]
Maybe a Hubble tension is the problem. All of the following has been proposed in previous postings.

$$
H = \sqrt{(1/8 \mbox{ or } 1/6 \mbox{ or } 1/3 \mbox{ or }  0.3883945571)\times 8\pi.G.\rho} \quad ; \quad H = (1 \mbox{ or } 2)/A
$$

where $G=$ gravitational constant, $\rho=$ mass density, $A=$ age.
Continuing with results from the above comment

$$
a = \gamma\frac{v^2}{s} \quad \mbox{with} \quad \gamma = 1.37 \mbox{ (boost factor)}
$$

it follows that

$$
v = \sqrt{\frac{a.s}{\gamma}}
$$

And so, with $H=2/A$ where $A=$ Age

$$
a = \frac{v^2}{s}+\frac{1}{2}H.v = \frac{a}{\gamma}+\sqrt{\frac{a.s}{\gamma}}/A \quad \Longrightarrow \quad A = \frac{\sqrt{a.s.\gamma}}{a(\gamma-1)}
$$

Numerically:
```
a := 10^(-9);
AU := 149597870700;
s := 5*1000*AU;
bf := 1.37;
v := sqrt(a*s/bf);
A := sqrt(a*s*bf)/(a*(bf-1));
y := 31556926;
A := A/y;
```
Resulting in a velocity = 739 m/s and an Age = 86,698 years. I don't (want to) know how absurd that is.

**Reply by AngelBlanco19** [reply]
> Someone has made a mistake somewhere ..

There are only two “parties”, the one that proves that matter is inexhaustible, infinite and eternal and the one that always postulates a Beginning of the Whole. There is no third way. They are self-exclusive, opposite and antagonistic, irreconcilable. This “news” is not new, it is at least about 2518 years old. So you choose, one path leads to light and the other to darkness.

Are you completely sure that you have chosen the right path?. If you had chosen the right path, you would not have encountered any of the alleged “problems” you mention. The matter is knowable and everything has an easy and logical explanation. Ockham's Razor works. If you want to know more about the right way and the solutions, just ask.

I already made some comments about it a few days ago, which were deleted, so I don't feel like writing much. I don't even know why I'm doing it. The ”mistakes" made are too old, too much. The key to everything will be given by the only solution of the Aporia «The Arrow» of the great genius of Zeno.

Time is not what our clocks mark, nor any of the drawings you exposes – it is the most important physical process of Matter, the basis of existence itself. It occurs only at a certain level of self-organization of Matter, creating everything finite, determinate and countable. Gravity is also not the phenomenon by which an apple can fall on our head, and from which it does not follow that we will have the Moon in the yard someday, in fact it moves away, just like we do from the Sun. Gravity is the second most important physical process of matter and its main function is the hot, permanent and eternal synthesis of chemical substances. The rest are side effects.

In case you are interested, I give you the EXACT values of several Universal Constants, Invariants  – they do not depend on ANYTHING.
Duration or extension of Time and Length of a Quantum of Action and a Phenomenon: 1 / ∞, is an infinitesimal magnitude, and therefore not determined and not countable.
EXACT Quantum of Time Value: 1•10^-44 s.
EXACT Quantum of Length Value: 1/6•10^-34 m.
EXACT Value of the Planck Time (duration of a complete cycle of any elementary particle of Matter, especially of a Photon, composed of 6 exact half cycles): 6 • 10^-44 s.
EXACT Value of the Frequency of our phenomenological Reality: 1 • 10^44 Hz.
Notes: 
1.- “Science" has long (I do not know how long) “has” (but does not know) the measured values of both the Quantum of Time and the Quantum of Length. I only give the EXACT values, which also serve to determine the measurement error.
2.- The Phenomenon is the result of the integration of 6 Quantum of Action, 3 Quantum of Time and Length after having received them. Each of the 6 Quantum of Action emitted at the same instant or infinitesimal moment of time, by any elementary particle of Matter, is the result of the differentiation of a Phenomenon, 3 Quantum of Time and Length after the Phenomenon has been integrated.
3.- Between two consecutive Quantum of Action, there is always a Quantum of Time and its corresponding Quantum of Length, and vice versa. The same thing between two consecutive Phenomena. It is what I have called the Dot-Dash-Dot system. A system of calculation for the conversion of the infinitesimal, indeterminate and non-countable, into finite, determinate and countable.

Good luck. The choice is in your hands and it only depends on you.

**Reply by Francois-Zinserling** [reply]
> I give you the EXACT values of several Universal Constants, Invariants – they do not depend on ANYTHING.
Duration or extension of Time and Length of a Quantum of Action and a Phenomenon: 1 / ∞, is an infinitesimal magnitude, and therefore not determined and not countable.
EXACT Quantum of Time Value: 1•10^-44 s.

No.

The exact value of 1 second is based on 1 rotation of the earth around its axis. Times by 60 to get 1 minute, times by 60 to get 1 hour, times by 24 to get the length of an average solar day. Days came first, obviously, then hours were defined (2 x 12 to separate day and night), then later minues and seconds when measuring things in 'days' and 'hours' was just not good enough anymore. 
This is from an earth day, not sidereal, so it's not even a proper rotation in absolute space. 
Nowadays based on the oscillation of cesium atoms, but that is still to try and make scientific what we initially chose as an 'earth second'.

The universe does not care about the earth's rotation being used in a universal constant. We're going to have a hard time explaining '1 second' to the aliens when they arrive, so it's not a universal constant.

Then there is time dilation, which you may want to prescribe to special relativity or not, but the time dilation from gravity is real and testable. 
So at a stretch there may be something like a 'local time' constant, which is a very big 'maybe', but it too is very very unlikely to be a nice round number like 1•10^-44 s.

**Reply by AngelBlanco19** [reply]
> The universe does not care about the earth's rotation being used in a universal constant.

Indeed, the Universe does not care much about what those of us who inhabit this planet think or stop thinking. What you mention is NOT the definition of a second and therefore is NOT a Universal Constant, nor can it be a Universal Invariant.

The **EXACT** values I have given must necessarily be "nice and EXACT”, as you call them, or else they could be neither Universal Constants nor Universal Invariants. As I wrote in my comment, Time is the **FUNDAMENTAL** physical process of Matter, and it is not a physical macro-process, it occurs and takes place only and exclusively within each elementary particle of Matter, of which we have 28 different types, including 4 different types of bosons and among these the Universal boson, the building block of **EVERYTHING** and the most common and widespread particle of inexhaustible, infinite and eternal Matter.

A second of Time, has exactly 1 • 10^44 consecutive infinitesimal Phenomena, each separated from each other, by their respective Quantum of Time of EXACT duration 1 • 10^-44 s, so a second of Time will always contain an indeterminate and non-countable part or portion (as in indefinite integrals). Every infinitesimal Phenomenon of our infinite and immobile phenomenological and therefore sensitive and not exact Reality can be integrated only and exclusively by elementary particles of Matter or elementary **UNITS** of existence, Noumena, Observers, Individuals - different denominations of the same physical essence, and never by an atom of chemical substance or higher forms of self-organization of Matter. That is why the physical process of Time, which is the basis of existence itself, only takes place and occurs at this level – at the level of an elementary particle of Matter. Absolutely everything finite, determinate and countable is a unique and exclusive product of the physical process of Time.

Moreover, surely you, like the vast majority, ignore the very essence of our infinite and immobile phenomenological and therefore sensitive and not exact Reality, as well as the essence of the Absolute that creates it. The Aporia «The Arrow» of the great genius of Zeno will give you the key to understanding this Absolute Truth.

Every infinite and immobile frame of our phenomenological and therefore sensitive and not exact Reality is the irrefutable proof that there is no dilation of either Time or Length, because absolutely all the positions of whatever, regardless of their relative state of rest, as well as absolutely all the intervals of Time and Length and absolutely all the distances, are determined **EXACTLY AND ABSOLUTELY**. The **EXACT** frequency of these infinite and immobile frames is 1•10^44 Hz. It is the best resolution of the best movie you will ever see - our Reality.

And Gravity, as I wrote, is the second most important physical process of Matter, and not the many side effects that we perceive from this physical process. Gravity "deforms” (is its natural state) the so-called “space-time”, but neither dilates Time, nor Length, forming Flat Spheres, a new concept. You should see these works. «The Gravitational Paradox and the Proposed Experiment to Demonstrate Gravitational Distortion of "Spacetime"» In English, Russian and Spanish versions, revised and expanded. 15 pages. 2 drawings. [https://vixra.org/abs/2202.0042](https://vixra.org/abs/2202.0042) and [https://github.com/orgs/a-cosmology-group/discussions/252#discussion-6983426](https://github.com/orgs/a-cosmology-group/discussions/252#discussion-6983426)
And «The Physical Mathematics and Geometry of Dialectical Materialism Versus the Euclidean "Mathematics" and "Geometry" of Philosophical Idealism» Spanish, Russian and English versions. 14 pages each. 14 drawings. [https://vixra.org/abs/2402.0116](https://vixra.org/abs/2402.0116) and [https://github.com/orgs/a-cosmology-group/discussions/253#discussion-6983433](https://github.com/orgs/a-cosmology-group/discussions/253#discussion-6983433)

And as I said in my previous comment, everyone must make their own decision - which path to choose.

If you have any doubts or disagree with something, do not hesitate to ask. Absolutely everything is demonstrable.

**Reply by HanDeBruijn** [reply]
@AngelBlanco19 .
You write in thread [126](https://github.com/orgs/a-cosmology-group/discussions/263#discussioncomment-10643051) :
>There are only two “parties”, the one that proves that Matter is inexhaustible, infinite and eternal and the one that always postulates a Beginning of the Whole. There is no third way. They are self-exclusive, opposite and antagonistic, irreconcilable. This “news” is not new, it is at least about 2518 years old. So you choose, one path leads to light and the other to darkness.

>Are you completely sure that you have chosen the right path?. If you had chosen the right path, you would not have encountered any of the alleged “problems” you mention. The Matter is knowable and everything has an easy and logical explanation. Ockham's Razor works. If you want to know more about the right way and the solutions, just ask.

And exactly the same text in [thread 263](https://github.com/orgs/a-cosmology-group/discussions/126#discussioncomment-10642773) .

So you keep the same file around and post it wherever you find it's appropriate ?

**Reply by AngelBlanco19** [reply]
> So you keep the same file around and post it wherever you find it's appropriate ?

Sorry, but if you read carefully you will see that it is not the same text, it is not a "copy & paste”, although the introductory part is the same, the rest of the body is not.

The introductory part is necessary and should always be the same, because a few days ago was deleted several comments that I had made, because it is not understood what it is that I expose, because it is NOT found in any textbook, or minimally “official” version of anything that I expose.

To understand what I am saying, you have to go back to Parmenides and Zeno, and as you should know, they are reviled by the official "science", by the current old paradigms.

Those who read the introductory part may know that nothing that I present will be minimally similar to what they are used to seeing and assume to “know”, they will know that these are two self-excluding, antagonistic and opposite “parties”. There are very weighty objective reasons, so that we can never understand each other and reach an agreement, a consensus. It's simply impossible. This is the reason why I must repeat the introductory part, so there will be no misunderstandings. I clarify and define my position, everyone should choose their own. I am open to dialogue, but I understand that it will be extremely difficult, for the objective reasons mentioned.

**Reply by AngelBlanco19** [reply]
> The introductory part is necessary and should always be the same,

The introductory part is necessary to draw attention to this **FUNDAMENTAL** problem, and let it be seen, that there is a viable solution, without having to create unnecessary entities and assumptions, without creating unsolvable logical contradictions. Anyway, Occam's Razor is working like never before in relation to this topic.

Each and every one of the problems mentioned in both topics (No. 126 and No. 263), and you mention several, have a simple and easy explanation, logical, without any rigging or forcing and therefore a definitive solution. Each piece fits into place naturally. But the reader should know, that what I expose is impossible within the framework that they usually work. The introductory part makes it clear. If I do not repeat it, the reader will not understand that I am in another “party” opposite perhaps to his own.

I even proposed to expose a single reason that justifies a thing like the Big Bang, because I can prove the falsity of each and every one of the supposed “reasons” put forward. No one has accepted the challenge. The introductory part makes it clear from which point of view the falsity of the current paradigms will be demonstrated. As I wrote in the introduction, a "party" leads to a dead end, to darkness. The evidence is everywhere and you seem to know several.

And as I said in my first comment, everyone must make their own decision - which path to choose.

**Comment by LifeIsStrange** [comment]
This new dataset that merge GAIA with Hubble images claims to disrupt astrometric accuracy, and therefore it might have the potential to improve tests of gravity
https://iopscience.iop.org/article/10.3847/1538-4357/ad5834
this should be used with panstars or euclid DR1 to achieve the biggest scientific breakthrough of the decade

other possible augmentations:
https://www.semanticscholar.org/paper/Connecting-VLBI-and-Gaia-Celestial-Reference-Frames-Malkin/c1474b0de6e37a2512c68ef645332c2528ae632b#citing-papers
https://www.semanticscholar.org/paper/Improvements-to-Pan-STARRS1-Astrometry-Using-Gaia-Lubow-White/308d83834346e0aca69af8d25a323f2d1c05778e
https://www.semanticscholar.org/paper/Improvements-to-Pan-STARRS1-Astrometry.-II.-for-White-Lubow/0d6048792f3629878e1d68e23068fc47bd5c63ae
I suppose DECAPS would be a nice alternative to panstarss
https://www.semanticscholar.org/paper/The-Dark-Energy-Camera-Plane-Survey-2-(DECaPS2)%3A-Saydjari-Schlafly/355d720a3afed1314c9bbd1e0581f13000589b07

FRBs will also allow to improve constraints on gravity
https://arxiv.org/abs/2409.11163

**Comment by LifeIsStrange** [comment]
New MOND paper just dropped:
https://tritonstation.com/2024/11/12/massive-galaxies-at-high-redshift-we-told-you-so/
https://iopscience.iop.org/article/10.3847/1538-4357/ad834d
https://news.ycombinator.com/item?id=42158130

**Reply by LifeIsStrange** [reply]
Upcoming MOND test
https://arxiv.org/abs/2411.17888

new cosmology? https://arxiv.org/abs/2411.18559

**Reply by HanDeBruijn** [reply]
Please stop the advertizing for MOND. Seven reasons why:
- [one](https://github.com/orgs/a-cosmology-group/discussions/70#discussioncomment-6265312)
- [two](https://github.com/orgs/a-cosmology-group/discussions/80#discussioncomment-6328995)
- [three](https://github.com/orgs/a-cosmology-group/discussions/111#discussioncomment-6517827)
- [four](https://github.com/orgs/a-cosmology-group/discussions/141#discussioncomment-6966770)
- [five](https://github.com/orgs/a-cosmology-group/discussions/148#discussioncomment-8724666)
- [six](https://github.com/orgs/a-cosmology-group/discussions/283#discussioncomment-11135443)
- deleted

**Reply by RedshiftDrift** [reply]
@HanDeBruijn MOND is a phenomenological model that has produced a lot of important data correlating galaxy rotation curves with other properties of galaxies.

@LifeIsStrange I encourage advertizing MOND since it produces data that support models.  It is emergent [from VM](https://github.com/orgs/a-cosmology-group/discussions/80#discussioncomment-6328995) *and also* STz.[^1]

[^1]: That's the topic of my next paper...
